doc/introduction/interfaces/numpy.rst

```diff
@@ -163,20 +163,22 @@ and two non-trainable arguments ``data`` and ``wires``:
         qml.CNOT(wires=[wires[0], wires[2]])
         return qml.expval(qml.PauliZ(wires[0]))
 
-We must specify that ``data`` and ``wires`` are NumPy arrays with ``requires_grad=False``:
+Since ``wires`` is a keyword argument, it will be automatically non-trainable. However, we
+must explicitly make ``data`` non-trainable by specifying it as a NumPy array with ``requires_grad=False``:
 
+>>> np.random.seed(42)  # make the results reproducable
 >>> weights = np.array([0.1, 0.2, 0.3])
 >>> data = np.random.random([2**3], requires_grad=False)
->>> wires = np.array([2, 0, 1], requires_grad=False)
->>> circuit(weights, data, wires)
-0.16935626052294817
+>>> wires = [2, 0, 1]
+>>> circuit(weights, data, wires=wires)
```

Comment:
As far as I am aware, this is the only code change necessary to make it work, the change above (`wires=None`) should not be needed